Agrega Celery

camilotorresf opened the pull request:
Para iniciar mi colaboración con el repositorio he agregado Celery, que es una herramienta que uso actualmente. Estaré agregando más de vez en cuando.

Commit:
Agrega Celery

satanas commented:
Hola @camilotorresf,

¿Crees que puedas agregar el texto en inglés? Pregunto porque el resto del README está en inglés y deberíamos mantener la consistencia. A menos que decidamos poner todo en español.

De resto veo todo bien.

Saludos

hernanramirez commented:
Otro detalle es que celery no es exclusivo de Django. Creo que debe ir en otro item como: asynchronous task y http://celery.readthedocs.org/en/latest/django/first-steps-with-django.html en la sección de django.

Procuremos tener las secciones lo mas puras posible..

satanas commented:
De acuerdo con @hernanramirez

Commit:
Change celery text to english and separate link for Django.

Use the english text from package description as pointed by J. Hernán Ramírez R. in the Google Group.
Separate celer into its own section: Backend development resources.
Put a separate link for Django pointing to the Django documentation section.

camilotorresf commented:
Hice unos cambios, por favor indicar si está bien así o requiere más ajustes.
Gracias.

satanas commented:
:shipit:

jgomo3 commented:
Bicho... a pesar de la unificación y demás, yo hubiera esperado que este recurso estuviera en español, por tratarse de algo con el sufijo -pyve.

satanas commented:
@jgomo3 Mi criterio de merge fue mantener la consistencia con el contenido que ya estaba. Si lo anterior estaba en inglés y lo nuevo quedaba en español tendríamos un arroz con mango.

Sin embargo, me parece totalmente válido tu comentario y creo que es algo que deberíamos conversar y definir para tener lineamientos claros para el futuro. Me parece que la lista es el lugar más indicado para esa discusión, así que abriré un nuevo hilo y ahí comentamos.

Saludos
